generated/nidmm/nidmm/session.py

```diff
@@ -1065,9 +1065,9 @@ def __init__(self, resource_name, id_query=False, reset_device=False, options={}
 
         -  Creates a new IVI instrument driver session and, optionally, sets the
            initial state of the following session properties:
-           range_check, QUERY_INSTR_STATUS,
-           cache, simulate,
-           record_coercions.
+           RANGE_CHECK, QUERY_INSTR_STATUS,
+           CACHE, simulate,
+           RECORD_COERCIONS.
```

Comment:
Do these show up as new "funky" warnings? At some point it'd be nice to go clean these up.

Reply:
Yes, since they are no longer in metadata. Before, they were in metadata as `'codegen_method': 'no'`, so they replacement worked, but it didn't actually exist in the generated files.